src/directions/utils.ts

```diff
@@ -68,7 +68,7 @@ export function buildRequest(
     payload = new URLSearchParams(formData);
   }
 
-  if (configuration.bearings) {
+  if (configuration.bearings && waypointsBearings) {
```

Comment:
Why exactly?

Reply:
Because `waypointsBearings` may be `undefined`